README.md

```diff
@@ -1,13 +1,17 @@
 # GitHub Tracker
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was about creating a GitHub tracker that displays my repositories, forked from Technigo. The objective for this project has been to continue practising JavaScript and in particular getting deeper into understanding API's (and for this project, using the GitHub documentation). This project contains my user data, repository data and a progress pie chart built with JS.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started to work on the project after I had done essential readings and video material provided by Technigo. I went through the project brief step by step and I found it very exciting to display the different kinds of data from the API's. But it has been quite challenging to get through this project, mainly in understanding how to write the code in Javascript (for example; when to declare a const, when to envoke a function, injecting HTML data etc) but new parts such as hiding files and getting acquainted with NPM has also been challenging and time consuming. Also learning about how to use tokens and API rate limits has given me some grey hair.
+
+But! Being finished with this project, that REALLY has had its ups and downs, in retrospect I know that I have learned A LOT. I'm obviously still on the surface when it comes to understanding everything and if I did have more time with this project I think I would dive into nesting fetches, displaying the number of pull requests, trying out other API endpoints, create a different chart and also try out styling charts in different ways.
+
+This was a really challenging yet fun and rewarding project that I feel that I have learned a lot from! 
```

Comment:
Great explanation on the problem in ReadMe! I'm inspired to write here more for myself